src/test/providers/videos/videos.provider.test.jsx

```diff
@@ -0,0 +1,337 @@
+import React, { useContext } from 'react';
+import { render, within, screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { act } from 'react-dom/test-utils';
+import VideosProvider, { VideosContext } from '../../../providers/videos';
+import {
+  setSearchQuery,
+  fetchVideosStart,
+  fetchVideosSuccess,
+  setVideoToWatch,
+  fetchVideosFailure,
+} from '../../../providers/videos/videos.actions';
+
+const videosMock = {
+  videoId1: {
+    etag: 'etag1',
+    videoId: 'videoId1',
+    image: '',
+    title: 'video title 1',
+    views: 'views 1',
+    timestamp: 'timestamp 1',
+    channel: {
+      channelId: 'chanelId 1',
+      title: 'channel title 1',
+      image: 'channel image1',
+      description: 'description 1',
+      subscribers: 1,
+    },
+  },
+  videoId2: {
+    etag: 'etag2',
+    videoId: 'videoId2',
+    image: '',
+    title: 'video title 2',
+    views: 'views 2',
+    timestamp: 'timestamp 2',
+    channel: {
+      channelId: 'chanelId 2',
+      title: 'channel title 2',
+      image: 'channel image 2',
+      description: 'description 2',
+      subscribers: 2,
+    },
```

Comment:
Same as before, I think this is the second time I've seen you use this data, defining it in fixture file will make things easier to maintain if the format of the data or other things change in the future